src/guides/guide_changi_short.md

```diff
@@ -1,4 +1,4 @@
-# Connect Metamask to Changi TestNet
+![252304387-ec0f54dc-908b-4a01-8a7c-e0e778ddef09](https://github.com/DeFiCh/handbook/assets/113342097/d5273de6-eba3-43ae-b8e7-1f39f0373830)# Connect Metamask to Changi TestNet
```

Comment:
Better to link the image locally instead

Reply:
done @shohamc1